server/blob/blob-api/src/main/java/org/apache/james/blob/api/DumbBlobStore.java

```diff
@@ -51,6 +51,7 @@
      * Save the blob with the provided blob id, and overwrite the previous blob with the same id if it already exists
      * The bucket is created if it not already exists.
      * This operation should be atomic and isolated
+     * Two blobs having the same blobId must have the same content
```

Comment:
note for later: we have a little issues with our comments:
- some are for people who will use the API
- some are for people who will implement the API
We should clearly differentiate between both